CHECK_POWER_COMMANDS/Backstop_File_Class.py

```diff
@@ -254,13 +257,19 @@ def insert_errors(self, lr_file, violations_list):
         outfile.writelines(ALR_lines)
         outfile.close()
 
-        # OK now copy the ACIS-LoadReview.txt.ERRORS file into ACIS-LoadReview.txt
-        try:
-            print('Copying ACIS-LoadReview.txt.ERRORS to ACIS-LoadReview.txt')
-            shutil.move('ACIS-LoadReview.txt.ERRORS', 'ACIS-LoadReview.txt')
-        except OSError as err:
-            print(err)
-            print('Examine the ofls directory and look for the .ERRORS file.')
+        # If the test flag was False, then copy the .ERRORS file to ACIS-LoadReview.txt
+        # If it was True then we leave the original ACIS-LoadReview.txt
+        # OK now copy the ACIS-LoadReview.txt.ERRORS file into ACIS- intact and keep
+        # The .ERRORS file for comparison
+        if not self.test_flag:
+            try:
+                print('Copying ACIS-LoadReview.txt.ERRORS to ACIS-LoadReview.txt')
+                shutil.move('ACIS-LoadReview.txt.ERRORS', 'ACIS-LoadReview.txt')
```

Comment:
BTW `shutil.move` moves (renames) the original file instead of copying. I'm not sure which is the desired operation, but the comments and logging message should be consistent with what is being done.

Reply:
Good catch on the comment/code mismatch.

Fixed